README.md

```diff
@@ -83,7 +83,7 @@ if __name__ == '__main__':
 | [azcaptcha.com](https://azcaptcha.com/) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Latin | ✅ |
 | [captcha.guru](https://captcha.guru/ru/reg/?ref=127872) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Latin | ✅ |
 | [cptch.net](https://cptch.net/auth/signup?frm=0ebc1ab34eb04f67ac320f020a8f709f) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Cyrillic/Latin | ❌ |
-| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Latin | ❌ |
+| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ❌ | Latin | ❌ |
```

Comment:
1. They do not have implemented the parameters to specify the details of the captcha of type image, but they do solve all the ones I mention.
2. Sorry. Since it is related to deathbycaptcha.com I included it here.

Reply:
Once again, deathbycaptcha does not allow to pass the values of these parameters.

Reply:
It is true that it does not allow to pass the parameters, but it does solve those types of images. I think you should clarify the table to which it corresponds, because at least I understand the types of captcha of image type that each service resolves and not the parameters that it accepts, for that I think that the documentation of each service is already there.